main.tf

```diff
@@ -16,25 +16,25 @@ provider "aws" {
 }
 
 module "orca" {
-  source = "./modules/orca"
-  database_app_user_pw = var.database_app_user_pw
-  default_tags = var.default_tags
-  prefix = var.prefix
-  subnet_ids = var.subnet_ids
-  database_port = var.database_port
-  postgres_user_pw = var.database_app_user_pw
-  database_name = var.database_name
-  database_app_user = var.database_app_user
-  ddl_dir = var.ddl_dir
-  drop_database = var.drop_database
-  platform = var.platform
-  lambda_timeout = var.lambda_timeout
+  source                         = "./modules/orca"
+  database_app_user_pw           = var.database_app_user_pw
+  default_tags                   = var.default_tags
+  prefix                         = var.prefix
+  subnet_ids                     = var.subnet_ids
+  database_port                  = var.database_port
+  postgres_user_pw               = var.database_app_user_pw
+  database_name                  = var.database_name
+  database_app_user              = var.database_app_user
+  ddl_dir                        = var.ddl_dir
+  drop_database                  = var.drop_database
+  platform                       = var.platform
+  lambda_timeout                 = var.lambda_timeout
   restore_complete_filter_prefix = var.restore_complete_filter_prefix
-  vpc_id = var.vpc_id
-  copy_retry_sleep_secs = var.copy_retry_sleep_secs
-  permissions_boundary_arn = var.permissions_boundary_arn
-  buckets = var.buckets
-  workflow_config = var.workflow_config
-  region = var.region
+  vpc_id                         = var.vpc_id
+  copy_retry_sleep_secs          = var.copy_retry_sleep_secs
+  permissions_boundary_arn       = var.permissions_boundary_arn
+  buckets                        = var.buckets
+  workflow_config                = var.workflow_config
+  region                         = var.region
```

Comment:
add the optional ORCA variables so they are bubbled up for the user.

-    sqs_delay_time = var.sqs_delay_time
-    maximum_message_size = var.maximum_message_size
-    sqs_message_retention_time = var.sqs_message_retention_time